tests/testthat/test-dataset.R

```diff
@@ -1,27 +1,50 @@
-context("Dataset object and methods") 
+context("Dataset object and methods")
 
 with(fake.HTTP, {
     test.ds <- loadDataset("test ds")
-    
+    today <- Sys.Date()
+    test_that("Dataset attributes", {
```

Comment:
Please give this block a non-duplicated name. And please move this to below the other `test_that("Dataset attributes". ...`